cypress.json

```diff
@@ -4,5 +4,7 @@
   "video": false,
   "reporterOptions": {
     "configFile": "mocha-multi-reporters.json"
-  }
+  },
+  "defaultCommandTimeout":	10000,
```

Comment:
es-dev-server is transforming cjs to es at startup which causes a delay => longer timeout